app/templates/main/map.html

```diff
@@ -0,0 +1,72 @@
+<html>
+<head>
+<script src="http://maps.googleapis.com/maps/api/js"></script>
+<script>
```

Comment:
I would put this in a separate .js file + folder for your static javascript, like in app/templates/js or app/static/js. Then include the script here with <script src ="path/to/file.js"></script>